chart/templates/cert-request-test-a2.yaml

```diff
@@ -0,0 +1,20 @@
+---
+apiVersion: verify.gov.uk/v1beta1
+kind: CertificateRequest
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/part-of: eidas-pki
+  name: verify-root-ca-test-a2
+  namespace: {{ .Release.Namespace }}
+spec:
+  countryCode: GB
+  commonName: Verify Test Root CA A2
```

Comment:
It's a distinct new root CA certificate for testing. 
The current roots CA for testing and production on the verify cluster are:
`verify-root-ca-production`
`verify-root-ca-test`
The names above correspond to old *RSA* keys in the HSM which are out of spec.
If you delete the secret derived from this `CertificateRequest` it will use the existing RSA key in the HSM.
To create an EC key we need to create a new `name` in the CertificateRequest.

I created an A1 CertificateRequest previously which created an HSM EC key with label `verify-root-ca-test-a1`, but the certificate did not have a subject identifier. I created A2 to generate a new clean cert with a subject identifier. I'm going to delete this file however, as if we delete the `verify-root-ca-test-a1` secret, the certificate should be recreated with a subject identifier.

Reply:
Discussed, with Karol - I'm going to delete this CertificateRequest and re-use CertificateRequest verify-root-ca-test-a1

Reply:
Okay thanks for the explanation Phil 👍